### DISCUSSION TITLE
Profile - {Add or Remove Channels conflict in default Profile}

### DISCUSSION THREAD

**Discussion by bachig26** [FreeTubeApp/FreeTube#1994] [General]
I have an issue regarding adding or removing a channel from a different Profile that reflects the default profile.
I would like to have a solution for this one.

Let's say, I have profiles like
1. Home (All Channels) => Default Profile
2. Music
3. News
4. ....

If I add a channel to the "Music" Profile, it correspondingly adds to the "Home" Profile, also removing this channel deletes in Default or Music Profile as it sync's all the time.

I need a **solution to have the SYNC in ONE-WAY and not BOTH WAYS or possibly an option to choose on the user end**.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
I also wanted to address this to the team. In my opinion there shouldn't be a All channel profile. Profiles should just reflect an individual profile. An individual account would mean that everything is separated from the other profiles for example channels, playlists, history and settings.

**Reply by bachig26** [reply]
> I also wanted to address this to the team. In my opinion there shouldn't be a All channel profile. Profiles should just reflect an individual profile. An individual account would mean that everything is separated from the other profiles for example channels, playlists, history and settings.

I don't know about the code part, but as I believe there can be a problem with syncing to the profile. and also as you have mentioned not everything is separated, there might need to be a rework in this part of the codes.
can this be added to the bug report?

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
I'll check with the team and if there is already a similar issue open first. I'll get back to u.

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
U can open up a report for the issue u mentioned. I'll start a discussion about the implementation of the all channel profile.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
> I have an issue regarding adding or removing a channel from a different Profile that reflects the default profile. I would like to have a solution for this one.
> 
> Let's say, I have profiles like
> 
>     1. Home (All Channels) => Default Profile
> 
>     2. Music
> 
>     3. News
> 
>     4. ....
> 
> 
> If I add a channel to the "Music" Profile, it correspondingly adds to the "Home" Profile, also removing this channel deletes in Default or Music Profile as it sync's all the time.
> 
> I need a **solution to have the SYNC in ONE-WAY and not BOTH WAYS or possibly an option to choose on the user end**.

@bachig26 I've looked over this again and im a little confused what the issue is ur facing. Could u please explain to me again.

**Reply by bachig26** [reply]
maybe I'll make a little clip and upload it soon.

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
we could also close ur issue if u think this willl solve it https://github.com/FreeTubeApp/FreeTube/issues/1996

**Reply by bachig26** [reply]
sure, I think you can close it, as it is the same. 👍
